python/enthic/extract_bundle.py

```diff
@@ -34,7 +34,7 @@ class ModifiedData(Enum):
 
 
 RE_DENOMINATION = compile(r'\s+|[\t\n]')  # NOT AN OBVIOUS PERFORMANCE GAIN...
-RE_POSTAL_CODE_TOWN = compile(r"([0-9]+)[ -]?([a-zA-Z0-9_ \'\"-\.\(\)\-]+)")
+RE_POSTAL_CODE_TOWN = compile(r"([0-9]+)[ -]?¨?([a-zA-Z0-9`ÀéÉèÈîÎ_ \'\"-\.\(\)\-]+)")
```

Comment:
This address parsing is related to fixing test_extract_bundle because in it, htere is one test that check "town" is never empty" but this regex produce empty town on some cases : when the town has an accent for example. That explain de second modification.
The first modification was made to handle one case I encountered : "75000  ¨Paris". The new regex extract 75000 and Paris correctly, the old one, no.

Reply:
> Please only request one feature at the time. This PR is mixing up a FIX and a Feature that is nowhere highlighted or explained. The master is already broken please focus on the fixes.

I don't know about what feature you're talking about. There is only fix and refactoring in this PR, and there are all mentionned in the PR comment.

Reply:
Hi @leonarf,

Unfortunately I tried to use this branch to generate the database and I got the following error:
```
Traceback (most recent call last):
  File "../python/enthic/extract_bundle.py", line 272, in <module>
    main()  # ONLY IF EXECUTED NOT WHEN IMPORTED
  File "../python/enthic/extract_bundle.py", line 268, in main
    process_daily_zip_file(join(CONFIG['inputPath'], file))
  File "../python/enthic/extract_bundle.py", line 198, in process_daily_zip_file
    code_confidentialite, info_traitement = read_identity_data(child, zipped_xml_name)
  File "../python/enthic/extract_bundle.py", line 157, in read_identity_data
    postal_code, town = read_address_data(identity, xml_file_name)
  File "../python/enthic/extract_bundle.py", line 99, in read_address_data
    raise Exception("In file {}. No town found in {}, maybe there is a special character that should be added to regex".format(xml_file_name, address_xml_item.text))
Exception: In file PUB_CA_809769086_7701_2015B00415_2016_3754.zip. No town found in 004 €, maybe there is a special character that should be added to regex
```